### PR TITLE
1949 toolbar test

### PR DESCRIPTION
**Pull request by Vtablos**

### CODE REVIEW

**Comment by mf-code** [comment]
[5731] - sv8yJmNO

**Comment by mf-code** [comment]
[5731] - nEEzTfet

**Comment by mf-code** [comment]
[5731] - yrj9cklw

**Comment by mf-code** [comment]
[5731] - 4LLLh2o1

**Comment by mf-code** [comment]
[5731] - 8ZIayll7

**Comment by mf-code** [comment]
[5731] - PhyoGOGY

**Comment by mf-code** [comment]
[5731] - 8GtBxMB5

**Comment by mf-code** [comment]
[5731] - 5BPspZyR

**Comment by mf-code** [comment]
[5731] - VBJZ44xh

**Comment by mf-code** [comment]
[5731] - u2LRMnz6

**Comment by mf-code** [comment]
[5731] - CFfQuGN1

**Comment by mf-code** [comment]
[5731] - v5eeiZdi

**Comment by mf-code** [comment]
[5731] - HcRtCpZ1

**Comment by mf-code** [comment]
[5731] - OLVL7ZRp

**Comment by mf-code** [comment]
[5731] - openjdk8 - Allure report on Netlify: https://5f0319a346302efdd6e1fa84--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5731] - openjdk10 - Allure report on Netlify: https://5f031ab574983f71089f4e09--musing-swartz-6fc060.netlify.app

File: jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/ToolbarSection.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    public static Text toolbarTextArea;
+
+    @UI("#toolbar-table")
+    public static Table toolbarTable;
+
+    @UI("#toolbar-table span:not(.example-spacer)")
+    public static JList<Text> toolbarRowsElementsWithText;
+}
+
+
```
Comment (gbuh) [inline jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/ToolbarSection.java:21]: Please delete extra line

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/common/ToolbarTests.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        toolbarTable.is().hasClass(classForTest);
+        toolbarRowsElementsWithText.is().values(listForTest);
+    }
+
+    @Test
+    public void multiRowToolbarColorTest() {
+        String colorForTest = "primary";
+        toolbarTable.has().attr("color", colorForTest);
+    }
+}
```
Comment (gbuh) [inline jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/common/ToolbarTests.java:46]: Please add a new line at the end of the file

**Comment by mf-code** [comment]
[5740] - KXIktq4O

**Comment by mf-code** [comment]
[5740] - 4RYyPrat

**Comment by mf-code** [comment]
[5740] - rweSVm6e

**Comment by mf-code** [comment]
[5740] - EHDwZGIL

**Comment by mf-code** [comment]
[5740] - 2o9bxgMy

**Comment by mf-code** [comment]
[5740] - WUSjLSJC

**Comment by mf-code** [comment]
[5740] - 7rSmXmyQ

**Comment by mf-code** [comment]
[5740] - IvTBhf8K

**Comment by mf-code** [comment]
[5740] - 2RxEoBeH

**Comment by mf-code** [comment]
[5740] - HU3CAnqR

**Comment by mf-code** [comment]
[5740] - guyL1MRP

**Comment by mf-code** [comment]
[5740] - yhVXHIcI

**Comment by mf-code** [comment]
[5740] - 0kcIqPeH

**Comment by mf-code** [comment]
[5740] - 7qbOBnp3

**Comment by mf-code** [comment]
[5740] - openjdk8 - Allure report on Netlify: https://5f05b20befa95803860bcc9d--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5740] - openjdk10 - Allure report on Netlify: https://5f05b2f3b760ea3813b3fbf3--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5745] - HDwME3Mx

**Comment by mf-code** [comment]
[5745] - owCB5mbR

**Comment by mf-code** [comment]
[5745] - I53DUuz0

**Comment by mf-code** [comment]
[5745] - kdqJn7yV

**Comment by mf-code** [comment]
[5745] - k51d1pdl

**Comment by mf-code** [comment]
[5745] - bwdQ4J5o

**Comment by mf-code** [comment]
[5745] - HePmz6l3

**Comment by mf-code** [comment]
[5745] - 2BhG3Hz4

**Comment by mf-code** [comment]
[5745] - TQB6TdBj

**Comment by mf-code** [comment]
[5745] - xF5IQDnS

**Comment by mf-code** [comment]
[5745] - zuXLMHQ3

**Comment by mf-code** [comment]
[5745] - H3o7lJNB

**Comment by mf-code** [comment]
[5745] - l1Et1Fu3

**Comment by mf-code** [comment]
[5745] - PadVDVxN

**Comment by mf-code** [comment]
[5745] - openjdk8 - Allure report on Netlify: https://5f05e46cdafb835bb711fa76--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5745] - openjdk10 - Allure report on Netlify: https://5f05e557d936a74e2effe048--musing-swartz-6fc060.netlify.app

**Pull request merged**